src/components/welcome/MyAccounts/PaginatedSafeList.tsx

```diff
@@ -0,0 +1,68 @@
+import type { ReactElement } from 'react'
+import { useMemo, useState } from 'react'
+import { Button, Box, Paper, Typography } from '@mui/material'
+import AccountItem from './AccountItem'
+import { type SafeItems } from './useAllSafes'
+import Track from '@/components/common/Track'
+import { OVERVIEW_EVENTS } from '@/services/analytics'
+import css from './styles.module.css'
+
+type PaginatedSafeListProps = {
+  safes: SafeItems
+  title: ReactElement
```

Comment:
```suggestion
  title: ReactNode
```

Then you can just pass a string.